data_pipeline/model/entities.py

```diff
@@ -0,0 +1,40 @@
+from data_pipeline.utils.xml_transform_util.timestamp import parse_timestamp
+
+
+class BaseEntity:
+
+    def __init__(self, data: dict):
+        self.data = data
```

Comment:
The `data` field is actually the dict / JSON itself. The other fields (`person_id`, `modified_timestamp`, etc.) were exposed as separate fields to help selecting the last version when using PostgreSQL. I don't think we need them as they are part of the data already. So really all entities here can be equivalent and merely serve as a different type (which might still be useful).

e.g.:
```python
class Person(BaseEntity):
    pass
```